ci/multibuild_config.sh

```diff
@@ -0,0 +1,6 @@
+# Define custom utilities
+# Test for OSX with [ -n "$IS_OSX" ]
+
+function run_tests {
+    tox
```

Comment:
This is going to build the shared library from source again, whereas you actually want to test the precompiled wheel (the one compiled inside the manylinux1 docker).
Use tox with —installpkg option, or just skip tox and run the tests directly. Multibuild already makes sure you are running the run_tests function from inside a fresh venv with the built wheel installed.
You can also lists tests requirements with an env variable